docs/installation_guides/Install.fr.md

```diff
@@ -59,18 +59,22 @@ git config --global --replace-all http.version HTTP/1.1
 ## Etape 7 : Installer Docker Devbox Vagrant
 
 - Utiliser un terminal en étant placé dans ```c:\workspace```
-- Cloner le repo github public du projet ```git clone https://github.com/gfi-centre-ouest/docker-devbox-vagrant docker-devbox```
+- Cloner le repo github public du projet ```git clone https://github.com/inetum-orleans/docker-devbox-vagrant docker-devbox```
 - Se placer dans le répertoire ```docker-devbox```
 - Copier-coller le fichier ```config.example.yaml``` en le renommant ```config.yaml``` et adapter les valeurs en fonction du besoin.
-Dans `env` vont se trouver des variables utilisées dans les scripts shell de provision de votre VM.
-Le tableau `provision_options` va servir à appeler des scripts supplémentaires présents dans `/provision/options`
+Dans `env` vont se trouver des variables utilisées dans les scripts shell de provision de votre VM. Le tableau `provision_options` va servir à appeler des scripts supplémentaires présents dans `/provision/options`
+
+Par exemple, ici, les variables AZ_* nous servent à nous connecter au dépôt Azure d'images Docker d'Inetum. Nous ajoutons l'installation de python et php directement dans la machine virtuelle via les scripts de provision optionnels.
 ```
 env:
   AZ_USERNAME: username Azure
   AZ_PASSWORD: password Azure
   AZ_TENANT: tenant Azure
-provision_options: ['mon-script']
+provision_options: ['python', 'php']
```

Comment:
```suggestion
provision_options: ['git-flow', 'azure-cli']
```
Installing python and php may mislead newcommers into thinking that they *need* that php script to work with php.